public/js/view.js

```diff
@@ -488,7 +444,15 @@ function view(){
 				clause1	 = geneName + changeOneCode
 				clause2 = geneName + '%5Btw%5D+AND+(' + changeOneCode + "+OR+" + changeThreeCode + ")",
 				changeQuery = clause1 + " OR (" + clause2 + ")",
-				changeHref = 'http://www.ncbi.nlm.nih.gov/pmc/?term=' + changeQuery;
+	    changeHref = 'http://www.ncbi.nlm.nih.gov/pmc/?term=' + changeQuery;
+	    
+		    createMutationHref = annotationsURL + '/annotations/save/mutation/?gene=' + geneName + 
```

Comment:
Looks good, but you can simplify greatly with [`jQuery.param()`](http://api.jquery.com/jQuery.param/).